server/planning/types/planning_featured.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+from pydantic import Field
+from typing import Annotated
+
+from .base import BasePlanningModel
+from superdesk.utc import utcnow
+from superdesk.core.resources import fields
+from superdesk.core.resources.validators import validate_data_relation_async
+
+
+class PlanningFeaturedResourceModel(BasePlanningModel):
+    date: datetime = Field(default_factory=utcnow)
+    items: list = Field(default_factory=list)
```

Comment:
What's it a list of?

Reply:
@MarkLark86 the previous schema had list defined as this with no specific schema hence why I left it as is: 

```"items": {"type": "list"},```

Reply:
I believe this should be a list of strings (the string IDs of the Planning items in it's list). We also have it in [Newshub as a list[str]](https://github.com/superdesk/newsroom-core/blob/async/newsroom/types/featured.py#L11). It would improve type checking etc